.github/workflows/publish.yaml

```diff
@@ -12,11 +12,11 @@ jobs:
       contents: read
       packages: write # allow GITHUB_TOKEN to publish packages
     steps:
-      - uses: actions/checkout@v4
       - name: Init nodejs
         uses: ./.github/actions/init-npm
         with:
-          node-version: 20.x
+          skip-build: false
```

Comment:
`node-version` defined as **required** on `action.yaml` file , i can tell it has a default value but how can we tell it is non breaking change?

Reply:
furthermore u left `node-version` with no changes within `pull_request.yml` file below -, maybe we should keep it here aswell?

Reply:
this is a publish job,the node version has nothing to do with the package after the code is built via `npm run build`

Reply:
Changing node version is a breaking change, even though nothing was broken.
The tests passed; it is the team leaders' responsibility to ensure full test coverage and to catch that.